nautobot/core/views/generic.py

```diff
@@ -185,11 +186,27 @@ def queryset_to_csv(self):
 
         return "\n".join(csv_data)
 
+    def validate_action_buttons(self):
+        """Verify actions in self.action_buttons are valid view actions. Raise exception if not valid"""
+
+        # ignore_actions are excluded because they are not view actions
```

Comment:
Can you elaborate on this? I know `export` is a special case of `list get`, but isn't `delete` a view action of its own?

Reply:
OK, so if `delete` isn't supposed to ever be in `action_buttons` then why do we need a specific check to ignore it? Should we instead fix any cases where `action_buttons` contains invalid actions?

Reply:
Yup, that's a better option